waves/src/theme.tsx

```diff
@@ -50,6 +42,16 @@ const theme = extendTheme({
                     color: "white",
                     _hover: {
                         bg: "blue.300",
+                        _disabled: {
```

Comment:
We had a _bug_ where the button disappeared when disabled and hovered over. Best would probably be to disable hovering if the button is disabled but I couldn't figure out how.

Reply:
Any reason why we are even overriding the `hover` styles?

To my knowledge, with all the styling in general, it is usually more consistent to modify the theme and stick to the pre-configured variants instead of inventing your own variants.

Reply:
It could be that I'm using the styles wrongly. The issue is that the default color is `grey` and hovering makes it darker. 

I wanted to change the background color (`bg`) to blue and that's where this weird behavior showed up, i.e. I had to set `hover` to a color as well, otherwise no hovering would happen and when the button is disabled the hovering changed the button color to white.